ooiui/static/css/common/scienceLayout.css

```diff
@@ -503,13 +510,87 @@ img#arrayImg {
 /*   background: white; */
 /* } */
 /*  */
+
+/**********/
+/* Scroll */
+/*********/
+
+table#popupInstrumentTable {
+  background-color: white;
+  width: 460px;
+  margin: 0px;
+  padding: 0px;
+}
+
+#popupInstrumentTable.nasdaq thead tr {
+  background: linear-gradient(#fff, #7E9CD9);
+  border-bottom-color: #F09349;
+}
+#popupInstrumentTable.nasdaq th {
+  text-transform: none;
+  font-weight: 500;
+  text-align: left;
+  background: linear-gradient(#fff, #7E9CD9);
+  background-image: -moz-linear-gradient(#fff, #7E9CD9);
+  background-image: -webkit-gradient(#fff, #7E9CD9);
+  border: 1px solid #D1DAF5;
+}
+
+#popupInstrumentTable.nasdaq th,
+#popupInstrumentTable.nasdaq td {
+  padding: 6px 8px;
+  font-size: 8pt;
+}
+
+#assembly-scroll-block.scrollable-block table {
+  margin-top: 2px;
+}
+
+#assembly-scroll-block.scrollable-block {
+  padding: 70px 0px;
+  margin-top: -72px;
+  margin-bottom: -72px;
+  margin-left: 0px;
+}
+
+#assembly-scroll-area.scrollable-area {
+  width: 100%;
+  max-height: 305px;
+  border: 2px solid #fff;
+  overflow-x: hidden;
+  overflow-y: auto;
+}
+
+
+#popupInstrumentTable.nasdaq tbody::-webkit-scrollbar {
+  width: 1px;
+  background: transparent;
+  border: 1px solid #D1DAF5;
+}
```

Comment:
these for sure

+#assembly-scroll-block.scrollable-block table {
+  margin-top: 2px;
+}
+
+#assembly-scroll-block.scrollable-block {
+  padding: 70px 0px;
+  margin-top: -72px;
+  margin-bottom: -72px;
+  margin-left: 0px;
+}
+
+#assembly-scroll-area.scrollable-area {
+  width: 100%;
+  max-height: 305px;
+  border: 2px solid #fff;
+  overflow-x: hidden;
+  overflow-y: auto;
+}